client/me/help/help-contact-us-footer.tsx

```diff
@@ -0,0 +1,41 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { Button, CompactCard, Gridicon } from '@automattic/components';
+import { HelpCenter } from '@automattic/data-stores';
+import { useDispatch as useDataStoreDispatch } from '@wordpress/data';
+import { useI18n } from '@wordpress/react-i18n';
+import type { FC } from 'react';
+
+import './style.scss';
+
+const HELP_CENTER_STORE = HelpCenter.register();
+
+const HelpContactUsFooter: FC = () => {
+	const { __ } = useI18n();
+	const { setShowHelpCenter, setInitialRoute } = useDataStoreDispatch( HELP_CENTER_STORE );
+
+	const onClick = () => {
+		recordTracksEvent( 'calypso_help_footer_button_click' );
+		setInitialRoute( '/contact-options' );
```

Comment:
This value will stick even after closing the Help Center. To reproduce:

1. Click "Contact support". 
2. Close the Help Center.
3. Go to any other page like My Home.
4. Open the Help Center, it will be stuck to contact options.

The fix is pretty simple, yield another action [here](https://github.com/Automattic/wp-calypso/blob/trunk/packages/data-stores/src/help-center/actions.ts#L61). 

```js
yield setInitialRoute ( undefined )
```

It may need some types adjustment [here](https://github.com/Automattic/wp-calypso/blob/trunk/packages/data-stores/src/help-center/actions.ts#L111).

Reply:
Oooh, nice catch, I missed that! FWIW, this will be partially fixed by https://github.com/Automattic/wp-calypso/pull/76893 (so that you'll never end up in a situation that the `Back` button in Help Center does not work). But I agree it should not get stuck in the sub-route like that.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/76894/commits/c81dc2305a2f602a0171e05ae7e0be99bfd00672